src/main/scala/org/camunda/feel/impl/parser/FeelParser.scala

```diff
@@ -114,10 +114,16 @@ object FeelParser {
     P(textualExpression.rep(1, sep = ",").map(s => ConstList(s.toList)))
 
   private def textualExpression[_: P]: P[Exp] =
-    P(expressionInParentheses | functionDefinition | forExpression | ifExpression | quantifiedExpression | expression2)
+    P(expressionWithContinuation | functionDefinition | forExpression | ifExpression | quantifiedExpression | expression2)
```

Comment:
🔧  While you're at it, perhaps also replace expression2 with disjunction directly. I believe that improves the understandability of the textualExpression.

Reply:
True. I'll change it in a further PR. I started to rewrite the structure of the parser and how the parser combinators are chained.